.github/workflows/create-release-candidate.yml

```diff
@@ -0,0 +1,345 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Create and upload release candidate artifacts
```

Comment:
can you make create rc and upload rc two separate jobs (single workflow, but two jobs with dependency). This is just in case that upload fails (due to network) we could retry just the upload.

In addition, I think there is a way to download artifacts from github action. it will be good to be able to download the rc manually if needed.

Reply:
Done, it is not separated as two jobs

Reply:
Done